tests/unit/s2n_security_policies_test.c

```diff
@@ -270,39 +270,39 @@ int main(int argc, char **argv)
         struct s2n_connection *conn = s2n_connection_new(S2N_CLIENT);
         s2n_connection_set_config(conn, config);
 
-        EXPECT_SUCCESS(s2n_connection_set_cipher_preferences(conn, "default"));
+        EXPECT_SUCCESS(s2n_connection_set_versioned_security_policy(conn, "default"));
```

Comment:
We still need to test `s2n_connection_set_cipher_preferences`, I would refrain from replacing and add additional unit tests for the new apis.

Reply:
This is outdated, not sure why its showing. The function isn't even named "versioned" anymore.
The cipher prefs are still being tested.